plugin.video.mlslive/addon.xml

```diff
@@ -17,5 +17,9 @@
     <forum>http://forum.xbmc.org/showthread.php?tid=158142</forum>
     <website>http://wiki.xbmc.org/index.php?title=Add-on:MLS_Live</website>
     <source>https://github.com/micahg/plugin.video.mlslive</source>
+    <news>v1.17.3
+- Marking as broken as MLS Live has shut down</news>
+  </extension>
```

Comment:
`</extension>` is invalid here

Reply:
lets hope our GSOC student can warn about these kind of issues through schema validation. They are not always easy to find.

Reply:
look at the travis log, don't know why this was merged with failed CI

Reply:
Fixed by lunatixz in https://github.com/xbmc/repo-plugins/pull/1875#merged-event